src/contexts/ExchangeIssuance/utils.ts

```diff
@@ -0,0 +1,94 @@
+import { IssuanceTradeType } from 'index-sdk/exchangeIssuance'
+import { IssuancePriceData } from './types'
+import {
+  dpiTokenAddress,
+  cgiTokenAddress,
+  indexTokenAddress,
+} from 'constants/ethContractAddresses'
+
+export const getIssuanceTradeType = (
+  isUserIssuing: boolean,
+  selectedCurrency: string,
+  IssuanceData: IssuancePriceData
+): IssuanceTradeType => {
+  if (isUserIssuing) {
+    if (
+      selectedCurrency !== 'wrapped_eth' &&
+      IssuanceData.trade_type === 'exact_out'
+    )
+      return IssuanceTradeType.ISSUE_EXACT_SET_FROM_TOKEN
+    else if (
+      selectedCurrency !== 'wrapped_eth' &&
+      IssuanceData.trade_type === 'exact_in'
+    )
+      return IssuanceTradeType.ISSUE_SET_FOR_EXACT_TOKEN
+    else if (IssuanceData.trade_type === 'exact_out')
+      return IssuanceTradeType.ISSUE_EXACT_SET_FROM_ETH
+    return IssuanceTradeType.ISSUE_SET_FOR_EXACT_ETH
+  } else if (selectedCurrency !== 'wrapped_eth')
```

Comment:
use braces around these else if statements. it's a bit hard to read and can be prone to error

Reply:
used braces